<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="convertBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>580</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QComboBox" name="methodBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>580</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="srcGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>510</width>
      <height>510</height>
     </rect>
    </property>
    <property name="title">
     <string>SRC</string>
    </property>
    <widget class="QLabel" name="srcImg">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>500</width>
       <height>500</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dstGroupBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>510</width>
      <height>510</height>
     </rect>
    </property>
    <property name="title">
     <string>DST</string>
    </property>
    <widget class="QLabel" name="dstImg">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>500</width>
       <height>500</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
